cdk/lib/security-hq.ts

```diff
@@ -1,17 +1,170 @@
+import {
+  ComparisonOperator,
+  Metric,
+  TreatMissingData,
+} from '@aws-cdk/aws-cloudwatch';
+import type { CfnTable } from '@aws-cdk/aws-dynamodb';
+import { InstanceClass, InstanceSize, InstanceType } from '@aws-cdk/aws-ec2';
+import type { CfnTopic } from '@aws-cdk/aws-sns';
 import { CfnInclude } from '@aws-cdk/cloudformation-include';
+import { Duration } from '@aws-cdk/core';
 import type { App } from '@aws-cdk/core';
+import { AccessScope, GuApplicationPorts, GuEc2App } from '@guardian/cdk';
+import { GuAlarm } from '@guardian/cdk/lib/constructs/cloudwatch';
 import type { GuStackProps } from '@guardian/cdk/lib/constructs/core';
-import { GuStack, GuStageParameter } from '@guardian/cdk/lib/constructs/core';
+import {
+  GuDistributionBucketParameter,
+  GuStack,
+} from '@guardian/cdk/lib/constructs/core';
+import {
+  GuDynamoDBReadPolicy,
+  GuDynamoDBWritePolicy,
+} from '@guardian/cdk/lib/constructs/iam';
 
+/**
+ * Migration steps:
+ *
+ * - ensure new stack is working okay
+ * - make DNS switch (defined in domains account)
```

Comment:
Any idea why the DNS is defined there? I had assumed it would reside in NS1...

Reply:
No idea to be honest.

Reply:
I had assumed Route53 could only manage entire zones and SHQ being a subdomain of `gutools.co.uk` means that NS1 manages it as `gutools.co.uk` is an NS1 managed zone 🤔.

Reply:
![Screenshot 2021-11-17 at 09 36 36](https://user-images.githubusercontent.com/858402/142175347-e36799d3-35b0-4443-ba6c-022b368302fa.png)